src/test/java/core/basesyntax/service/filehandler/ReportGeneratorTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.filehandler;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.exceptions.EmptyStorageException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ReportGeneratorTest {
+    private Storage storage;
+    private ReportGenerator generator;
+
+    @BeforeEach
+    void setUp() {
```

Comment:
I think generator and storage init can be in Before all method, but clearing DB can be in this method